lms/djangoapps/ccx/views.py

```diff
@@ -349,6 +349,9 @@ def visit(node, depth=1):
         widgets, which use text inputs.
         """
         for child in node.get_children():
+            # in case the children are visible to staff only, skip them
```

Comment:
There's (a little) more detail in https://github.com/mitocw/edx-platform/issues/82 : Course authors don't want coaches to use re-use blocks in their CCX that were marked "hide from students" in Studio. They usually contain content that isn't meant to be viewed by students.